.formatter.exs

```diff
@@ -1,6 +1,6 @@
-
+# Used by "mix format"
```

Comment:
Let's remove this, `.formatter.exs` is standard in all Elixir projects so no need to specify this here.

```suggestion
```